aule/visitors/alfaASTVisitor.py

```diff
@@ -48,43 +44,64 @@ def visitArrayExpression(self, ctx):
     def visitBinaryOperator(self, ctx):
         return ctx.getText()
 
+    def visitSetOperator(self, ctx):
+        return ctx.getText()
```

Comment:
Need `.upper()` here to be same with `IN` and `NOT`

Reply:
done